### ISSUE TITLE
Solid 1.6 spread/deproxification behaviour breaking solid-dnd drag handlers

### ISSUE THREAD

**Issue by martinpengellyphillips** [solidjs/solid#1328]
### Describe the bug

The changes in Solid 1.6 handling of spread/deproxification break previously working assignement of event listeners to elements in solid-dnd.

This code now fails to update the element with the appropriate event listeners once they are defined:

```js
const Draggable = () => {
  const draggable = createDraggable(1);
  return (
    <div
      ref={draggable.ref}
      style={transformStyle(draggable.transform)}
      class="draggable-container"
    >
      <div class="content">Draggable</div>
      <div class="handle" {...draggable.dragActivators}>  // <--- this no longer works as before
        ✪
      </div>
    </div>
  );
};
```

`draggable.dragActivators` looks like
```js
dragActivators: {
  enumerable: true,
  get: () => {
    return draggableActivators(id, true);
  },
},
```

What complicates things is that the activators depend on sensors being registered elsewhere, which is why this has to be reactive. This worked fine in Solid 1.5.x. 

### Your Example Website or App

https://github.com/thisbeyond/solid-dnd-site

### Steps to Reproduce the Bug or Issue

1. Clone https://github.com/thisbeyond/solid-dnd-site
2. Install all dependencies with `npm install` (ensuring you have solid-js@1.6.1)
3. Run with `npm run dev`
4. Try the `fine grained / drag handle` example. Note it does not work (no drag behaviour).
5. Downgrade solid-js to 1.5.7 and note the same example now works.

### Expected behavior

The drag activators should update reactively as sensors are registered.

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by LiQuidProQuo** [comment]
https://playground.solidjs.com/?hash=-985355670&version=1.5.7
```html
<div class="handle" hello="world">✪ DIV</div>
```

https://playground.solidjs.com/?hash=-1779828763&version=1.6.1
```html
<div class="handle">✪ DIV</div>
```

**Comment by ryansolid** [comment]
@LiQuidProQuo `f` is a function.. and should be called as such. The fact it worked before was unintentional.

@martinpengellyphillips clear your package.lock and get your babel-preset-solid version to 1.6. And the example works again.

**Comment by martinpengellyphillips** [comment]
@ryansolid thanks - confirming that fixes the issue after I force installed 1.6.1 of `babel-preset-solid`. 

Any idea why `babel-preset-solid` 1.6.x series is not installed by default (even after removing the package lock file)? Is there anything I can do as `solid-dnd` owner to help folks avoid hitting this issue themselves?
